src/components/cards/collaborations/AddCollaboratorButton.jsx

```diff
@@ -0,0 +1,232 @@
+    potentialCollaborator?.full_name?.toLowerCase().trim() ?? '';
+  if (fullName.includes(lowerCaseSearchTerm)) return true;
+
+  const email =
+    potentialCollaborator?.email?.toLowerCase().trim() ?? '';
+  if (email.includes(lowerCaseSearchTerm)) return true;
+
+  return false;
+}
+
+export default function AddCollaboratorButton({
+  userData,
+  mutation,
+}) {
+  const intl = useIntl();
+  const userGuid = userData?.guid;
+
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [selectedUserGuid, setSelectedUserGuid] = useState(null);
+
+  const {
+    data: users,
+    isLoading: isUsersLoading,
+    error: getUsersError,
+  } = useGetUsers();
+
+  const {
+    mutate: addCollaboration,
+    isLoading: isAddCollaborationLoading,
+    error: addCollaborationError,
+    reset: resetAddCollaborationMutation,
+  } = mutation;
+
+  const error = addCollaborationError || getUsersError;
+
+  function handleCloseDialog() {
+    setIsDialogOpen(false);
+    setSearchTerm('');
+    setSelectedUserGuid(null);
+    resetAddCollaborationMutation();
+  }
+
+  function handleSubmitSearchTerm(e) {
+    e.preventDefault();
+
+    const form = e.target;
+    const formData = new FormData(form);
+    const searchValue = formData.get('search');
+
+    setSearchTerm(searchValue);
+    setSelectedUserGuid(null);
+  }
+
+  function handleSubmitCollaborator() {
+    addCollaboration(
+      { userGuid: selectedUserGuid },
+      { onSuccess: handleCloseDialog },
+    );
+  }
+
+  const currentCollaboratorGuids = (
+    userData?.collaborations || []
+  ).map(collaboration => {
+    const memberGuids = Object.keys(collaboration?.members || {});
+    return memberGuids.find(guid => guid !== userGuid);
+  });
+
+  const columns = [
+    { name: 'name', labelId: 'NAME', align: 'left' },
+    { name: 'email', labelId: 'EMAIL', align: 'left' },
+    { name: 'role', labelId: 'ROLE', align: 'left' },
+    {
+      name: 'created',
+      labelId: 'CREATED',
+      align: 'left',
+      options: {
+        cellRenderer: cellRendererTypes.date,
+        cellRendererProps: { fancy: true },
+      },
+    },
+  ];
+
+  const tableData = (users || [])
+    .filter(user =>
+      filterCollaborator(
+        user,
+        searchTerm,
+        userGuid,
+        currentCollaboratorGuids,
+      ),
+    )
+    .map(user => ({
+      guid: user?.guid,
+      email: user?.email,
+      name: user?.full_name,
+      created: user?.created,
+      role: intl.formatMessage({ id: getHighestRoleLabelId(user) }),
+    }));
+
+  return (
+    <>
+      <StandardDialog
+        titleId="ADD_COLLABORATION"
+        open={isDialogOpen}
+        onClose={handleCloseDialog}
+        fullWidth
+        maxWidth="md"
+      >
+        <DialogContent
+          style={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: 12,
+          }}
+        >
+          {users && (
+            <form
```

Comment:
I'm curious why a normal form element here.

Reply:
What would you use instead?

Reply:
I don't know. I see FormControl more often in the code base. Mostly I was hoping to learn whether it was a conscious choice to not use FormControl and why.

Reply:
FormControls are wrappers that provide functionality for inputs. The TextField that I am using in this form "is a complete form control". The form could have multiple FormControls, one for each input. I did intentionally use a form because I think it is semantically correct and because it provides some nice behavior by default.
https://v4.mui.com/components/text-fields/#textfield

I picked TextField because that is what was used in IndividualSelector.